### DISCUSSION TITLE
How to use postgres ltree with blaze persistence

### DISCUSSION THREAD

**Discussion by h-mergel** [Blazebit/blaze-persistence#1678] [Q&A] [answered]
Is it possible to write postres ltree queries with blaze persistence?

I am having trouble transforming this query to criteria API :
`select * from some_table where path @> '1.3.4'`

**Comment by jwgmeligmeyling** [comment, marked as answer]
Yes, it is possible, but you'll need to register a custom function that renders `{0} @> {1}`. How to register custom functions can be read in the documentation. You can also use Hibernates methods to register custom functions. This obviously assumes you have already successfully mapped your attribute to the `ltree` datatype, which might require the introduction of a custom type as well.
